website/static/js/pages/project-dashboard-page.js

```diff
@@ -93,6 +94,32 @@ var institutionLogos = {
     }
 };
 
+/* Send with ajax calls to work with api2 */
+var xhrconfig = function (xhr) {
+    xhr.withCredentials = window.contextVars.isOnRootDomain,
+    xhr.setRequestHeader('Content-Type', 'application/vnd.api+json;');
+    xhr.setRequestHeader('Accept', 'application/vnd.api+json; ext=bulk');
+};
+var categoryList = [];
+// Load categories to pass in to create project
+var loadCategories = function _loadCategories () {
```

Comment:
Minor: Rather than pushing to a "global" array (`categoryList`), it would be cleaner for `loadCategories` to return a promise that resolves to a list of categories. The resulting API would look like:

```js
loadCategories().then(function(categoriesList) {
    // do stuff with categoriesList
})
```

Mithril has a m.deferred object that would be useful for this: http://mithril.js.org/mithril.deferred.html .

Certainly not a blocker, though.